Sources/XcodeProj/Project/XcodeProj.swift

```diff
@@ -157,33 +211,36 @@ extension XcodeProj: Writable {
     ///   If true will remove all existing schemes before writing.
     ///   If false will throw error if scheme already exists at the given path.
     public func writeSchemes(path: Path, override: Bool = true) throws {
-        guard let sharedData = sharedData else { return }
+        let sharedDataPath = XcodeProj.sharedDataPath(path)
+        try sharedData?.writeSchemes(path: sharedDataPath, override: override)
 
-        let schemesPath = XcodeProj.schemesPath(path)
-        if override, schemesPath.exists {
-            try schemesPath.delete()
-        }
```

Comment:
This is still happening, only now as part of [`XCSharedData.writeSchemes`]( https://github.com/tuist/XcodeProj/pull/739/files#diff-d9813515fcdca420ec369da92701017d37747feec7abd4519c9cc008e06a3796R72)